esp/src/eclwatch/nls/es/hpcc.js

```diff
@@ -178,10 +187,13 @@ define(
     Downloads: "Descargas",
     DownloadToCSV: "Bajar en formato CSV",
     DropZone: "Zona de carga",
+    DueToInctivity: "Se desconectará de todas las sesiones de ECL Watch en 3 minutos debido a la inactividad.", 
```

Comment:
I would take out "la"
```suggestion
    DueToInctivity: "Se desconectará de todas las sesiones de ECL Watch en 3 minutos debido a inactividad.", 
```